feature/experimental/policy/policy_vrf_selection/otg_tests/policy_based_traffic_steering_default_to_nondefault/README.md

```diff
@@ -0,0 +1,311 @@
+    B2[Port2]
+    B3[Port3]
+end
+subgraph ATE2 [ATE2]
+    C2[Port2]
+    C3[Port3]
+end
+A1[ATE1:Port1] <-- IBGP(ASN100) --> B1; 
+B2 <-- EBGP(ASN100:ASN200) --> C2;
+B3 <-- EBGP(ASN100:ASN300) --> C3;
+```
+### Advertisements:
+   * ATE1:Port1 advertises following prefixes over IBGP to DUT:Port1.
+     * IPv4Prefix1/24 IPv6Prefix1/64
+     * IPv4Prefix2/24 IPv6Prefix2/64
+     * IPv4Prefix3/24 IPv6Prefix3/64
+     * IPv4Prefix4/24 IPv6Prefix4/64
+     * IPv4Prefix5/24 IPv6Prefix5/64<br>
+
+   * ATE2:Port2 and DUT:Port2 from EBGP neighborship between each other over on respective AFI-SAFI connections and advertise the following prefixes
+     * From ATE2:Port2 to DUT:Port2
+       * IPv4Prefix6/24 IPv6Prefix6/64
+       * IPv4Prefix7/24 IPv6Prefix7/64
+       * IPv4Prefix8/24 IPv6Prefix8/64
+       * IPv4Prefix9/24 IPv6Prefix9/64
+       * IPv4Prefix10/24 IPv6Prefix10/64
+     * From DUT:Port2 to ATE2:Port2
+       * All the IBGP learnt prefixes from Prefix1 to Prefix5<br>
+
+   * ATE2:Port3 and DUT:Port3 from EBGP neighborship between each other over on respective AFI-SAFI connections and advertise the following prefixes
+     * From ATE2:Port2 to DUT:Port2
+       * IPv4Prefix6/24 IPv6Prefix6/64
+       * IPv4Prefix7/24 IPv6Prefix7/64
+       * IPv4Prefix8/24 IPv6Prefix8/64
+       * IPv4Prefix9/24 IPv6Prefix9/64
+       * IPv4Prefix10/24 IPv6Prefix10/64
+     * From DUT:Port2 to ATE2:Port2
+       * All the IBGP learnt prefixes from Prefix1 to Prefix5<br>
+
+
+     * **[Important: Please note]** Following advertisements must happen only during the test one at a time as instructed<br>
+
+* Configure the DUT to honor the TOS byte of destination 
+| Match Condition                           | Outer Header DSCP Set to: |
+|-------------------------------------------|---------------------------|
+| DSCP value BE1, Destination IPv4Prefix6/24"                         | 3                         |
+| DSCP value BE1, Destination IPv6Prefix6/64"                         | 3                         |
+| DSCP value AF1, Destination IPv4Prefix7/24"                         | 11                        |
+| DSCP value AF1, Destination IPv6Prefix7/64"                         | 11                        |
+| DSCP value AF2, Destination IPv4Prefix8/24"                         | 19                        |
+| DSCP value AF2, Destination IPv6Prefix8/24"                         | 19                        |
+| DSCP value AF3, Destination IPv4Prefix9/24"                         | 27                        |
+| DSCP value AF3, Destination IPv6Prefix9/64"                         | 27                        |
+| DSCP value AF4, Destination IPv4Prefix10/24"                       | 35                        |
+| DSCP value AF4, Destination IPv6Prefix10/24"                       | 35                        |
+
+
+
+   * DUT has 2 VRFs, Default and Non-Default. Configure Route leaking from the Default to the Non-default VRF to share reachability between VRFs.<br>
```

Comment:
Can we clarify to say that we are expecting all routes that exist in the default VRF to be imported into the non-default VRF?

Reply:
Actually, I think this test has a bug -- or it's not immediately clear to me how you're expecting this to work.

I would expect that the way to test this is that we have:

 * the default VRF with one set of routes to ATE port 1
 * the non-default VRF with a different set of routes to ATE port 2

We change our VRF selection policy and we determine where the packets are getting routed with the VRF selection policy based on that.

This is essentially how it's tested elsewhere (see https://github.com/openconfig/featureprofiles/tree/main/feature/experimental/policy/policy_vrf_selection/otg_tests/protocol_dscp_rules_for_vrf_selection_test).

In the case that's defined here:

 * we're introducing BGP (why?)
 * we're leaking all routes between the two VRFs -- so if the default VRF selects a route then the non-default VRF does the same. so later on we say something about how eBGP selects a route, but there's no definition of why that happens given that we're leaking routes between the two?

Reply:
Had to change the complete README, appreciate if you can take a look on the latest updated README.